content/en/database_monitoring/setup_sql_server/gcsql.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Setting Up Database Monitoring for Google Cloud SQL managed SQL Server
+kind: documentation
+description: Install and configure Database Monitoring for SQL Server managed on Google Cloud SQL
+further_reading:
+- link: "/integrations/sqlserver/"
+  tag: "Documentation"
+  text: "Basic SQL Server Integration"
+- link: "/database_monitoring/troubleshooting/?tab=sqlserver"
+  tag: "Documentation"
+  text: "Troubleshoot Common Issues"
```

Comment:
This doesn't exist yet, so recommend removing until it does. Plus we just moved all the troubleshooting info to DB specific pages, so it likely won't be at this URL path anyway.


```suggestion

```

Reply:
Good catch, agreed, thanks!